__tests__/react/type.js

```diff
@@ -208,4 +208,17 @@ describe("userEvent.type", () => {
       expect(onKeyUp).not.toHaveBeenCalled();
     }
   );
+
+  it.each(["input", "textarea"])("should focus on type", type => {
+    const onFocus = jest.fn();
+    const { getByTestId } = render(
+      React.createElement(type, {
+        "data-testid": "input",
+        onFocus: onFocus
```

Comment:
nit: could use a shorthand for consistency with the rest of the test file

```suggestion
        onFocus
```